bindings/python/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+CMAKE_MINIMUM_REQUIRED (VERSION 2.6)
+
+INCLUDE (${CMAKE_SOURCE_DIR}/cmake/python.cmake)
+
+PROJECT (state_observation_py)
+
+# Dependencies.
+FINDPYTHON ()
+ADD_REQUIRED_DEPENDENCY ("eigenpy >= 1.3.0")
+
+# Set the include directories.
+INCLUDE_DIRECTORIES ("include/")
+INCLUDE_DIRECTORIES (${PYTHON_INCLUDE_DIR})
+INCLUDE_DIRECTORIES (${PYTHON_INCLUDE_DIR2})
+
+# Create library.
+FILE (GLOB_RECURSE ${PROJECT_NAME}_SOURCES "src/*.cpp")
+SET (${PROJECT_NAME}_LIB ${PROJECT_NAME})
+ADD_LIBRARY (${${PROJECT_NAME}_LIB} SHARED ${${PROJECT_NAME}_SOURCES})
+TARGET_LINK_LIBRARIES (${${PROJECT_NAME}_LIB} ${PYTHON_LIBRARY} boost_python state-observation)
```

Comment:
Here, you must not linked against Python for the shared library. Its must be done by the Python interpreter directly. Please use TARGET_LINK_BOOST_PYTHON for that purpose.